packages/seacas/libraries/ioss/src/Ioss_EntityBlock.C

```diff
@@ -79,3 +79,44 @@ Ioss::Property Ioss::EntityBlock::get_implicit_property(const std::string &my_na
 
   return Ioss::GroupingEntity::get_implicit_property(my_name);
 }
+
+bool Ioss::EntityBlock::equal(const Ioss::EntityBlock &rhs)
+{
+  /* COMPARE element topologies */
+  if( *(this->topology_) != *(rhs.topology_) ) {
+    return false;
+  }
+
+  if( this->idOffset != rhs.idOffset ) {
+    return false;
+  }
+
+  return Ioss::GroupingEntity::equal( rhs );
+}
+
+bool Ioss::EntityBlock::operator==(const Ioss::EntityBlock &rhs) const
+{
+  /* COMPARE element topologies */
+  if( *(this->topology_) != *(rhs.topology_) ) {
+    printf("EntityBlock: TOPOLOGY mismatch\n");
```

Comment:
Do not output stuff from operator==

Reply:
Addressed in branch `tkordenbrock-seacas/add-ioshell-compare`.